### PR TITLE
Create welcome screens

### LINKED ISSUE #48: I see a series of welcome screens
__Wireframe link__

- Therapist: https://www.figma.com/file/CcYmhfnXreAPxlfyEmGsAH/Chiltern-Music-Therapy?node-id=1063%3A67
- Client: https://www.figma.com/file/CcYmhfnXreAPxlfyEmGsAH/Chiltern-Music-Therapy?node-id=1063%3A3

**Note: This doesn't include the profile page for the therapist, that's covered in issue #47 **

---

### Acceptance Criteria:

- [x] Create therapist welcome screens in line with wireframes
- [x] Create client welcome screens in line with wireframes

### PR DESCRIPTION
**Pull request by thejoefriel**
#### Issues this PR relates to
Closes #48 

=======

_NOTE: REMEMBER NOT TO REVIEW UNLESS ALL ACCEPTANCE CRITERIA HAS BEEN TICKED OFF_

---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist

Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)

- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

File: client/src/pages/General/Welcome/style.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import styled from '@emotion/styled';
+
+export const Wrapper = styled.div`
+  padding-top: ${({ theme: { spacings } }) => spacings[9]};
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+
```
[the remark this answers is not in the document]
Comment (thejoefriel) [reply client/src/pages/General/Welcome/style.js:9]: done

File: client/src/pages/General/Welcome/style.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  ${({ theme }) => theme.media.tablet} {
+    padding-top: 0;
+    flex-grow: 1;
```
[the remark this answers is not in the document]
Comment (thejoefriel) [reply client/src/pages/General/Welcome/style.js:12]: done

File: client/src/pages/General/Welcome/index.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    }
+
+    return navRoutes.THERAPIST.PROFILE;
+  };
+
+  return (
+    <S.Wrapper>
+      <Title {...decideContent().title} mbM="3" />
+      <Row mb="6" mbM="4">
+        <Col w={[4, 8, 8]}>
```
[the remark this answers is not in the document]
Comment (thejoefriel) [reply client/src/pages/General/Welcome/index.js]: done

File: client/src/pages/General/Welcome/index.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+          <T.P color="gray8">{decideContent().text}</T.P>
+        </Col>
+      </Row>
+
+      <S.ButtonWrapper>
+        <Col w={[4, 6, 6]}>
```
[the remark this answers is not in the document]
Comment (thejoefriel) [reply client/src/pages/General/Welcome/index.js]: done

File: client/src/components/Title/index.js [hunk 5]
```diff
@@ -1,9 +1,16 @@
...
+  boldSection,
+  boldFirst,
+  lightSection2,
+  mb,
+  mbM,
+}) =>
   boldFirst ? (
-    <Row mb="5">
+    <Row mb={mb || '5'} mbM={mbM || '5'}>
       <Col w={[4, 8, 6]} style={{ display: 'flex ' }}>
```
[the remark this answers is not in the document]
Comment (thejoefriel) [reply client/src/components/Title/index.js]: done

**Review by RamyAlshurafa — APPROVED** [review]

**Pull request merged**